examples/async.rs

```diff
@@ -10,17 +10,16 @@
 
 extern crate igd;
 extern crate futures;
-extern crate tokio_core;
+extern crate tokio;
 
 use igd::async::search_gateway;
 use igd::PortMappingProtocol;
 use futures::future::Future;
 
 fn main() {
-    let mut evloop = tokio_core::reactor::Core::new().unwrap();
-    let handle = evloop.handle();
+    let f = futures::lazy(|| {
```

Comment:
Why is `lazy` required here?

Reply:
it _might_ not be in this example, but generally it's required so tokio::spawn occurs within an existing context. see [here](https://tokio.rs/docs/futures/spawning/)